### ISSUE TITLE
为什么多了一个   pool = pool + l(pool)

### ISSUE THREAD

**Issue by henbucuoshanghai** [yuhuan-wu/P2T#12]
pool = pool + l(pool)

**Comment by henbucuoshanghai** [comment]
不要这一代码  不好吗？

**Comment by yuhuan-wu** [comment]
Do you mean the relative positional encoding (RPE) in our pooling-based attention? RPE has negligible computational cost and can bring significant improvement both on image classification and semantic segmentation as shown in our ablation study.

You can refer to our paper for more details. Thanks.

**Comment by henbucuoshanghai** [comment]
&nbsp;l(pool) 的l是self.d_convs  为什么是四个组卷积呢？
self.d_convs1 = nn.ModuleList([nn.Conv2d(embed_dims[0], embed_dims[0], kernel_size=3, stride=1, padding=1, groups=embed_dims[0]) for temp in pool_ratios[0]])

**Comment by yuhuan-wu** [comment]
It is because we have 4 different pooling ratios corresponding to 4 pooled maps in implementation.
